hpe3parclient/client.py

```diff
@@ -4130,3 +4130,48 @@ def getSnapshotsOfVolume(self, cpgName, volName):
                 if volume['copyOf'] == volName:
                     snapshots.append(volume['name'])
         return snapshots
+
+    def createSchedule(self, schedule_name, volume_name, tasksch, snaprefix='sv-', expiration = None, retain = None, readonly=False):
```

Comment:
Create schedule functions have to be generic and not specific to snapshot.

createSchedule(self, schedule_name, task)
deleteSchedule(self, schedule_name)

Pack all the values in the consuming application (ansible)